aixplain/factories/metric_factory.py

```diff
@@ -97,13 +97,13 @@ def get(cls, metric_id: Text) -> Metric:
         return metric
 
     @classmethod
-    def list(cls, model_id: Text=None, source_required: bool=None, reference_required: bool=None, page_number: int = 0, page_size: int = 20,) -> List[Metric]:
+    def list(cls, model_id: Text=None, is_source_required: Optional[bool]=None, is_reference_required: Optional[bool]=None, page_number: int = 0, page_size: int = 20,) -> List[Metric]:
```

Comment:
Set the default to False:

```
is_source_required: bool = False
is_reference_required: bool = False
```

Reply:
By default we need to not have any filter. So None means it'll show both metrics where src is required and where it isn't. Only if the user sends True or False do we actually filter. We can choose to keep a default but the user then might not be aware of this